RuntimeMocks/drivers_config.h

```diff
@@ -0,0 +1,68 @@
+/**@file
+ * This file is part of the Leon3 Serial CCSDS Driver for the Test Environment.
```

Comment:
Test Environment -> TASTE Leon3 Drivers. This is just cosmetics... but I think it will look better.